### ISSUE TITLE
Support Standalone Batch Norm

### ISSUE THREAD

**Issue by mcr229** [pytorch/executorch#11586]
### 🚀 The feature, motivation and pitch

Currently XNNPACK doesn't support Batch Norm, however, we can generally support it by fusing it with convolution before hand. However for standalone batch norms we can also convert them into depthwise conv1ds. Take a look at the following code here:
https://github.com/pytorch/executorch/blob/main/backends/xnnpack/_passes/fuse_batch_norm_with_conv.py

**Comment by ayushrajdahal** [comment]
I'd like to work on this.

**Comment by mcr229** [comment]
Assigning it to you! Feel free to communicate in the discord for progress/updates and blockers!

**Comment by Mat-thias** [comment]
Hi @mcr229 @ayushrajdahal 
Has this been fixed?
I would love to give it a go, I recently worked on a project on static quantisation and I needed to fold the batchnorm layers into the proceeding linear/conv layers. 
I can assign it to myself?
